packages/flutter/lib/src/widgets/web_image.dart

```diff
@@ -0,0 +1,143 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../../widgets.dart';
+import '_web_image_io.dart' if (dart.library.js_interop) '_web_image_web.dart'
+    as impl;
+
+/// A substitute for [Image.network] that follows the web's [same origin policy][].
+///
+/// On mobile this widget delegates to [Image.network]. On the web this widget
```

Comment:
```suggestion
/// On non-web platforms this widget delegates to [Image.network]. On the web this widget
```

Don't forget desktop :)

Reply:
Done.